modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/DataEngineLayoutBuilderHandler.es.js

```diff
@@ -75,6 +101,8 @@ export default function ({defaultLanguageId, namespace}) {
 			dataLayout.description = description;
 			dataLayout.name = name;
 
+			clearNameInputIfNeeded();
+
```

Comment:
Well, I don't know if it would be the best solution to do this here, I think this validation should be on the backend, checking if the pages are translated or if it is too expensive to compare since it can be a huge computing cost, it would be Frontend not to send the language at the time of save, probably those who take care of that are the internals of the DE...

Another option where this can avoid being saved would be in the component that handles this behavior, in the case of the title or someone listening to this translation manager event to copy the default title value to the new language, in this case, we should remove this behavior. If that's what I understand, this function only clears the title when the language is different from the default, right? Maybe this change should be in the `LocaleChangedHandler`?

Reply:
Hi @matuzalemsteles! This validation already exists on the backend, however, we think this validation should also be on the frontend because of how the input localized works. I don’t believe that it has a great cost, since the only thing that is done is emptying the input if this lenguaje is not among the translated language or is different from the default language.

Furthermore, this check should only be done when the page is publisher.  For that reason this check is done in `DataEngineLayoutBuilderHandler` (in `saveDDMStructure`) and not `LocaleChangedHandler`, which is executed every time a language is selected.